backend/routes/project.js

```diff
@@ -216,6 +217,7 @@ router.post(
           proposal.signature = req.body.signedMessage;
           proposal.round = currentRoundNumber;
 
+          cacheSpecificProposal(airtableRecordId);
```

Comment:
Shouldn't we cache after Proposal is saved to db, before we send the success response?  Does it make sense caching it event if it's not save to db?

Reply:
It is saved to airtable

Reply:
So it doesn't cause any issues not being saved inside the db, we are only getting it from Airtable?

Reply:
It's confusing because we are sending 400 if it's not save to db, but then it would be saved

Reply:
I guess we should leave it like this but open an issue because the bug wasn't created by this PR. Like before proposal would be saved in Airtable but if db save crashes on the frontend we get an error and say that proposal was not created